### PR TITLE
Improve intro

### PR DESCRIPTION
**Pull request by akaszynski**
Working on intro.

Adding a video like https://streamlit.io/ shortly.

### CODE REVIEW

**Review by tkoyama010 — APPROVED** [review]
LGTM

File: tutorial/02_mesh/README.rst [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+===============
 
 Learn about the PyVista data types and how you can use it to visualize almost anything. (15 min for talk, 10 min for exercise)
 
 .. tip::
 
-    This section of the tutorial was adopted from `What is a mesh? <https://docs.pyvista.org/user-guide/what-is-a-mesh.html>`_
-    chapter of the User Guide in the PyVista documentation.
+uide/what-is-a-mesh.html>`_
+    chapter of the User Gui
```
Comment (banesullivan) [inline tutorial/02_mesh/README.rst:11]: This is causing the build to fail

**Review by banesullivan — APPROVED** [review]
Nice work!

File: tutorial/02_mesh/README.rst (hunk 1 above)
Comment (banesullivan) [inline tutorial/02_mesh/README.rst:11]:
```suggestion
    This section of the tutorial was adopted from `What is a mesh? <https://docs.pyvista.org/user-guide/what-is-a-mesh.html>`_
    chapter of the User Guide in the PyVista documentation.
```

**Comment by akaszynski** [comment]
Don't merge quite yet, adding video.

**Comment by akaszynski** [comment]
Please validate videos @banesullivan.

**Pull request merged**